mssql-plugin/src/e2e-test/java/io.cdap.plugin/common.stepsdesign/TestSetupHooks.java

```diff
@@ -16,76 +16,292 @@
 
 package io.cdap.plugin.common.stepsdesign;
 
+import com.google.cloud.bigquery.BigQueryException;
+import io.cdap.e2e.utils.BigQueryClient;
 import io.cdap.e2e.utils.PluginPropertyUtils;
 import io.cdap.plugin.MssqlClient;
 import io.cucumber.java.After;
 import io.cucumber.java.Before;
 import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import stepsdesign.BeforeActions;
 
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.sql.SQLException;
+import java.util.NoSuchElementException;
+import java.util.UUID;
 
 /**
  * Mssql test hooks.
  */
 public class TestSetupHooks {
+  @Before(order = 1)
+  public static void setTableName() {
+    String randomString = RandomStringUtils.randomAlphabetic(10).toUpperCase();
+    String sourceTableName = String.format("SOURCETABLE_%s", randomString);
+    String targetTableName = String.format("TARGETTABLE_%s", randomString);
+    PluginPropertyUtils.addPluginProp("sourceTable", sourceTableName);
+    PluginPropertyUtils.addPluginProp("targetTable", targetTableName);
+    String schema = PluginPropertyUtils.pluginProp("schema");
+    PluginPropertyUtils.addPluginProp("selectQuery", String.format("select * from %s.%s", schema,
+                                                                   sourceTableName));
+  }
 
-    @Before(order = 1)
-    public static void setTableName() {
-        String randomString = RandomStringUtils.randomAlphabetic(10).toUpperCase();
-        String sourceTableName = String.format("SOURCETABLE_%s", randomString);
-        String targetTableName = String.format("TARGETTABLE_%s", randomString);
-        PluginPropertyUtils.addPluginProp("sourceTable", sourceTableName);
-        PluginPropertyUtils.addPluginProp("targetTable", targetTableName);
-        String schema = PluginPropertyUtils.pluginProp("schema");
-        PluginPropertyUtils.addPluginProp("selectQuery", String.format("select * from %s.%s", schema,
-                sourceTableName));
-    }
 
-    @Before(order = 2, value = "@MSSQL_SOURCE_TEST")
-    public static void createTables() throws SQLException, ClassNotFoundException {
-        MssqlClient.createSourceTable(PluginPropertyUtils.pluginProp("sourceTable"),
-                PluginPropertyUtils.pluginProp("schema"));
-        MssqlClient.createTargetTable(PluginPropertyUtils.pluginProp("targetTable"),
-                PluginPropertyUtils.pluginProp("schema"));
-    }
+  @Before(order = 2, value = "@MSSQL_AS_SOURCE")
+  public static void createSourceTables() throws SQLException, ClassNotFoundException {
+    MssqlClient.createSourceTable(PluginPropertyUtils.pluginProp("sourceTable"),
+                                  PluginPropertyUtils.pluginProp("schema"));
```

Comment:
nit: `sourceTable`, `targetTable`, `schema` strings have been across this file. It's a good idea to create constants for these strings.

Reply:
Done.